streams/src/test/java/org/apache/kafka/streams/integration/ResetIntegrationTest.java

```diff
@@ -343,15 +356,13 @@ private void cleanGlobal(final String intermediateUserTopic) {
             parameters = new String[]{
                 "--application-id", APP_ID + testNo,
                 "--bootstrap-servers", CLUSTER.bootstrapServers(),
-                "--zookeeper", CLUSTER.zKConnectString(),
```

Comment:
Maybe we can add one test, that makes sure we don't fail if `--zookeeper` is specified? -> `doNotFailIfZookeeperIsSpecified` ?

Reply:
Good enough to add the `--zookeeper` args back in?

Reply:
This should, do. Maybe only add it back for one case so we cover both. And add a comment why we do this.

Reply:
Ack